soda_machine.rb

```diff
@@ -1,18 +1,25 @@
 class SodaMachine
   attr_reader :sodas, :cash
+  attr_accessor :sold_soda
```

Comment:
Is `attr_accessor` needed here? Be careful not to over expose your information unnecessarily.